src/scss/_objects.resources-tile.scss

```diff
@@ -0,0 +1,12 @@
+.resources-tile {
+  background-color: #fff;
+
+  &__thumbnail {
+    width: 100px;
+    height: 100px;
```

Comment:
Is it possible to use an existing Grommet variable for these, such as `$graphic-small-size` (96px)?